assets/src/edit-story/components/library/panes/media/dropDownMenu.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useState, useRef } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import DropDownList from '../../../../components/form/dropDown/list';
+import Popup from '../../../../components/popup';
+import { ReactComponent as More } from '../../../../icons/more_horiz.svg';
+
+const MoreIcon = styled(More)`
+  height: 28px;
+  width: 28px;
+`;
+
+const IconContainer = styled.div`
+  position: absolute;
+  top: 8px;
+  right: 8px;
+`;
+
+/**
+ * Get a More icon that displays a dropdown menu on click.
+ *
+ * @param {Object} props Component props.
+ * @param {boolean} props.showDisplayIcon If the More icon should be displayed
+ * @param {function(boolean)} props.menuCallback Callback for when menu is opened / closed.
+ * @return {null|*} Element or null if should not display the More icon.
+ */
+function DropDownMenu({ showDisplayIcon, menuCallback }) {
+  const options = [
+    { name: __('Edit', 'web-stories'), value: 'edit' },
+    { name: __('Delete', 'web-stories'), value: 'delete' },
+  ];
+
+  const [showMenu, setShowMenu] = useState(false);
+  const moreRef = useRef();
```

Comment:
`useState(!showDisplayIcon)`? (or `useState(!collapsed)`, as per above comment)

Reply:
This should always be initialised as `false`, as the menu does not show by default (until the icon is clicked on).

Reply:
Makes me think whether a storybook story for this area is useful. I feel like it could be useful for testing.

Reply:
Added one! (Also added a bunch of test assets, not sure if this is the best place for them to live.)